machineconfiguration/v1/doc.go

```diff
@@ -0,0 +1,7 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+
+// +groupName=machineconfiguration.config.openshift.io
```

Comment:
this group doesn't match the group in your golang code.  Which do you mean?

Reply:
this one here is wrong